.gitmodules

```diff
@@ -3,8 +3,9 @@
 	url = https://github.com/scipy/scipy-mathjax.git
 [submodule "scipy/sparse/linalg/_propack/PROPACK"]
 	path = scipy/sparse/linalg/_propack/PROPACK
-	url = https://github.com/scipy/PROPACK
+	url = https://github.com/thalassemia/PROPACK.git
 	shallow = true
+	branch = wrapped_blas
```

Comment:
Do we want to merge https://github.com/thalassemia/PROPACK/commit/cd4e3d3bfc7d1f667111076adc7bccb3e23d0a71 into https://github.com/scipy/PROPACK first?

Reply:
Yes, we should merge that and bump the submodule commit in this PR.

Reply:
Open PR for xref: https://github.com/scipy/PROPACK/pull/7